rotkehlchen/chain/evm/transactions.py

```diff
@@ -541,6 +550,53 @@ def ensure_tx_data_exists(
         tx_receipt = self.dbevmtx.get_receipt(cursor, tx_hash, self.evm_inquirer.chain_id)
         return tx_data, tx_receipt  # type: ignore  # tx_data can't be None here
 
+    def get_and_ensure_internal_txns_of_parent_in_db(
+            self,
+            tx_hash: 'EVMTxHash',
+            to_address: 'ChecksumEvmAddress',
+            for_address: 'ChecksumEvmAddress',
```

Comment:
is this always a tx `for_address` sending to `to_address`. The naming is confusing. Maybe `from_address` and `to_address`?